### DISCUSSION TITLE
Github oauth is always 404

### DISCUSSION THREAD

**Discussion by Arinji2** [pocketbase/pocketbase#3989] [Q&A] [answered]
My code is 
`   const authData = await pb
          .collection("users")
          .authWithOAuth2({ provider: "github" });`
          
I added the client id and secret into pocketbase
![image](https://github.com/pocketbase/pocketbase/assets/87296817/41358f01-d375-4bc4-a202-dc03de44b86a)
![image](https://github.com/pocketbase/pocketbase/assets/87296817/25fc93fe-85eb-4c14-84a2-cf8f9ec6909e)

**Comment by ganigeorgiev** [comment, marked as answer]
Double check for typos and spaces in the client id and secret fields.

From the second screenshot the `%20` character at the beginning of the `client_id` param indicates that maybe you copied with a space.

**Comment by Arinji2** [comment]
Yup, client_id had a space in it, thanks :D
